FirebaseMessaging/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # unreleased -- v.7.0.0
 - [changed] Remove the deprecated FCM direct channel API and Upstream send API. (#6430)
+- [changed] The `messaging:didReceiveRegistrationToken:` should be able to return a null token. Update the API parameter fcmToken to be nullable. (#6647)
```

Comment:
Prefer issue number over PR number when possible for changelogs.

Reply:
Done.